cps/tasks/metadata_extract.py

```diff
@@ -100,6 +104,9 @@ def _send_shelf_title(self):
         except Exception as e:
             log.error("An error occurred during the shelf title sending: %s", e)
 
+    def _ignore_shorts(self, requested_urls):
+        requested_urls = {url: requested_urls[url] for url in requested_urls.keys() if requested_urls[url]["duration"] > 60}
+    
```

Comment:
This function defaults playlists and channels downloads to ignoring Shorts. Like constants MAX_VIDEOS_PER_DOWLOAD and MAX_GB_PER_DOWNLOAD, we can make it configurable by the user to toggle it on/off.

Reply:
> Like constants MAX_VIDEOS_PER_DOWNLOAD

Should the option to ignore "Shorts" be part of this PR, if it's straightforward to implement safely?

Reply:
Moving it into another PR might make it more feature-relevant than portraying as a fix. Zero confusion is better.

Reply:
> Moving it into another PR might make it more feature-relevant than portraying it as a fix. Zero confusion is better.

Please link to any PR or ticket explaining what should happen next:

How should people who want "YouTube Shorts" included when downloading channels/playlists/etc set that option at the command-line — prior to clicking "Download to IIAB" ?